### PR TITLE
Update section name for teams and add title to start UI

### PR DESCRIPTION
**Pull request by typfel**

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/ConversationList/SearchView/SearchViewController.m [hunk 1]
```diff
@@ -23,53 +23,87 @@
 #import "IconButton.h"
 #import "WireSyncEngine+iOS.h"
 #import "WAZUIMagicIOS.h"
+#import "UIColor+WAZExtensions.h"
 
-@interface SearchViewController ()
+@interface SearchViewController () <ZMUserObserver>
 @property (nonatomic, readwrite) PeopleInputController *peopleInputController;
-@property (nonatomic, readwrite) SearchView *searchView;
+@property (nonatomic, readwrite) NSString *searchTitle;
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/ConversationList/SearchView/SearchViewController.m]: I think the `UIViewController` has an integrated property called `title` of type `NSString`, so I guess this one can be used?

File: Wire-iOS/Sources/UserInterface/ConversationList/SearchView/SearchViewController.m [hunk 2]
```diff
@@ -79,4 +113,13 @@ - (void)onCloseButtonPressed:(id)sender;
     }
 }
 
+#pragma mark - ZMUserObserver
+
+- (void)userDidChange:(UserChangeInfo *)note
+{
+    if ([note.user isSelfUser] && note.accentColorValueChanged) {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/ConversationList/SearchView/SearchViewController.m]: Can probably be done through classy without the observation?

Comment (typfel) [inline Wire-iOS/Sources/UserInterface/ConversationList/SearchView/SearchViewController.m]: Maybe, just moved the code.

Comment (typfel) [inline Wire-iOS/Sources/UserInterface/ConversationList/SearchView/SearchViewController.m]: using `accent-current-color` doesn't update the view the accent color changes.

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/ConversationList/SearchView/SearchViewController.m]: Sounds like a bug in our Classy implementation. Let's merge it as is then!

**Pull request merged**